docs/application/dotnet/guides/TOC.md

```diff
@@ -13,16 +13,26 @@
 ## [Package Manager](app-management/package-manager.md)
 
 # [Natural User Interface](nui/NUIoverview.md)
-## [NUI Hello World Tutorial](nui/hello-world.md)
+## [Setting Up the NUI Development Environment on Ubuntu](nui/setup-ubuntu.md)
+## [Getting Started](nui/hello-world.md)
+## [Layer](nui/layer.md)
 ## [UI Components](nui/ui-components.md)
+### [Styling UI Components](nui/styling-controls-with-JSON.md)
 ### [Visuals](nui/visuals.md)
-### [Button](nui/button.md)
+### [View](nui/view.md)
+### [Buttons](nui/button.md)
 ### [FlexContainer](nui/flexcontainer.md)
 ### [ImageView](nui/imageview.md)
+### [ScrollView](nui/scrollview.md)
+### [Slider](nui/slider.md)
+### [TableView](nui/tableview.md)
 ### [TextLabel](nui/textlabel.md)
-### [CustomView](nui/creating-custom-view-controls.md)
```

Comment:
@Seoyeon2Kim creating-custon-view-controls.md file is missing on the restructured TOC.md. Is it intended?

Reply:
Oh.. I'm sorry I missed it. I'll change to add it right now.

Reply:
@Seoyeon2Kim It's ok.. : ) Your update in #160 is also applied.